config-example.json

```diff
@@ -11,6 +11,7 @@
   },
   "db_path": "./ripple-rest.db",
   "max_transaction_fee": 1000000,
+  "proxy": "http://username:password@0.0.0.0:3128",
```

Comment:
Sorry, missed this the first time, but I think this should be commented out so that the example config will work without changes.